dev/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0/tck/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i guess you'll want the tck at some point but it should be in it's own project, not nested